CMakeLists.txt

```diff
@@ -898,6 +898,19 @@ add_custom_target(lrelease
 ## Miscellaneous Microsoft Visual Studio settings
 ##
 if (MSVC)
+   # Add a post-build event to deploy the application to the output folder. Same as building the INSTALL target.
+   STRING(TOUPPER ${CMAKE_GENERATOR} UPPER_CMAKE_GENERATOR)
+   if (${UPPER_CMAKE_GENERATOR} MATCHES "VISUAL STUDIO")
+      # We want $(Configuration) passed verbatim. Note the round brackets.
+      set(CONFIG_STR "$(Configuration)")
+
+      add_custom_command(
+         TARGET MuseScoreStudio POST_BUILD
```

Comment:
I believe the target is named `mscore` on this branch (see main/CMakeLists.txt)

Reply:
🤦 of course

Reply:
That's still not working though